src/app/treasure-hunt/treasure-hunt-report/opportunity-summary.service.ts

```diff
@@ -140,9 +140,23 @@ export class OpportunitySummaryService {
 
   
   getIndividualOpportunitySummary(thOpportunity: TreasureHuntOpportunity, settings: Settings): OpportunitySummary {
+    let oppSummary: OpportunitySummary;
     let results: TreasureHuntOpportunityResults = this.getCalculatorTreasureHuntResults(thOpportunity, settings);
     let opportunityMetaData: OpportunityMetaData = this.getOpportunityMetaData(thOpportunity.opportunitySheet);
-    let oppSummary: OpportunitySummary = this.getNewOpportunitySummary(opportunityMetaData, results);
+    if (thOpportunity.opportunityType === Treasure.waterHeating){
+      let waterHeatingOpportunity = thOpportunity as WaterHeatingTreasureHunt;
```

Comment:
Again with future mixed opp calculators we could think about adding an 'isMixed' property to TreasureHuntOpportunity. and offload some of the logic from 146-155 to a separate method that handles all the mixed calcs.